src/carbonable/erc3525/library.cairo

```diff
@@ -295,6 +299,10 @@ namespace ERC3525 {
     func __mint_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
double `_` prefix?

Reply:
Yes, I took the same terminology as in the Solidity implementation. I guess a doubly '__' private method can *only* be called inside a single '_'  private method.